src/main/java/roomescape/advice/RestExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package roomescape.advice;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import roomescape.exception.BadRequestException;
+import roomescape.exception.ConflictException;
+import roomescape.exception.NotFoundException;
+
+@RestControllerAdvice(annotations = RestController.class)
+public class RestExceptionHandler {
+    @ExceptionHandler(NotFoundException.class)
+    public ResponseEntity<String> notFound(NotFoundException e) {
+        return ResponseEntity.status(404).body(e.getMessage());
```

Comment:
```suggestion
        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
```
상태코드를 숫자코드가 아니라 이미 정의된 Enum을 활용해보면 어떨까요?

Reply:
오 좋은 것 같아요! 바로 반영했습니다!

[refactor(RestExceptionHandler): 상태 코드를 enum으로 변경](https://github.com/woowacourse/spring-roomescape-admin/pull/319/commits/3af69e9e2a827d492545fac60ba8fa877899dd31)